### ISSUE TITLE
Add Braxton the Bully to Mission Files

### ISSUE THREAD

**Issue by bleehu** [bleehu/CXDocs#181]
Add the Braxton the Bully canned mission to the mission files tool.

**Comment by bleehu** [comment]
Blocked by PR #180

**Comment by bleehu** [comment]
Closing as won't do. Will re-open when a system for reading files out of adjacent repos exists.
